modules/solid_mechanics/src/neml2/utils/NEML2Utils.C

```diff
@@ -54,6 +55,21 @@ namespace NEML2Utils
 
 #ifdef NEML2_ENABLED
 
+neml2::Model &
+get_model_moose(const std::string & mname,
+                MooseBase * moose_base,
+                bool enable_ad,
+                bool force_create)
+{
+  neml2::OptionSet extra_opts;
+  extra_opts.set<MooseApp *>("_MooseApp") = &moose_base->getMooseApp();
```

Comment:
```suggestion
  extra_opts.set<MooseApp *>("_moose_app") = &moose_base->getMooseApp();
```
to be consistent with neml2 naming conventions.